### PR TITLE
add description about en-gb instead of en-us

### PR DESCRIPTION
**Pull request by ilaredo**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-9748](https://jira-dc.paloaltonetworks.com/browse/CIAC-9748)

## Description
When using rasterize from a German server, pages are translated, and a popup appears. The rasterize integration needs to allow language selection.
user should user --accept-lang=en-gb instead of --accept-lang=en-us

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/rasterize/Integrations/rasterize/README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ If you are using the integration to rasterize un-trusted URLs or HTML content, s
     ```
 
     To set a language for the browser, add the *--accept-lang* argument followed by the desired language code in IETF BCP 47 format. For example, `--accept-lang=de-DE`.
+If you're willing to set language to en-US please use en-GB instead.
```
Comment (ShirleyDenkberg) [inline Packs/rasterize/Integrations/rasterize/README.md:32]:
```suggestion
If you want to set the language to en-US, use en-GB instead.
```

File: Packs/rasterize/Integrations/rasterize/rasterize_description.md [hunk 2]
```diff
@@ -12,5 +12,6 @@ If you are using the integration to rasterize un-trusted URLs or HTML content, s
     --disable-auto-reload,[--disable-dev-shm-usage]
     ```
     To set a language for the browser, add the *--accept-lang* argument followed by the desired language code in IETF BCP 47 format. For example, `--accept-lang=de-DE`.
+    If you're willing to set language to en-US please use en-GB instead.
```
Comment (ShirleyDenkberg) [inline Packs/rasterize/Integrations/rasterize/rasterize_description.md:15]:
```suggestion
    If you want to set the language to en-US, use en-GB instead.
```

**Comment by ShirleyDenkberg** [comment]
@tkatzir @eyalpalo Doc review completed.  
Do you need to add release notes for this change?

**Review by tkatzir — APPROVED** [review]

**Review by eyalpalo — APPROVED** [review]

**Comment by ilaredo** [comment]
We discovered an issue when selecting 'en-us' in the Chrome options. It appears that the default language is 'en-us' and the Chrome options support languages other than 'en-us'. As a temporary solution, the customer can select 'en-gb' to obtain the English translation. Additionally, the documentation has been updated to inform users that if they wish to use 'en-us', they should select 'en-gb' instead.

**Pull request merged**